routers/review.js

```diff
@@ -0,0 +1,110 @@
+const { Router } = require("express");
+const BooksCollection = require("../models").booksCollection;
+const Review = require("../models").review;
+const Book = require("../models").book;
+const User = require("../models").user;
+const Collection = require("../models").collection;
+
+const auth = require("../auth/middleware");
+
+const router = new Router();
+
+router.post("/post", auth, async (req, res) => {
+  const { reviewTitle, reviewContent, bookId, collectionId } = req.body;
+  if (!reviewTitle && !reviewContent) {
+    return res
+      .status(400)
+      .send({ message: "Please provide review title and content" });
+  }
+  try {
+    await Review.create({
+      title: reviewTitle,
+      content: reviewContent,
+      likes: 0,
+    });
+
+    const latestReview = await Review.findOne({
+      order: [["id", "DESC"]],
+    });
```

Comment:
is this latestReview the one you just created? why not then do:
```
const latestReview = await Review.create({
      title: reviewTitle,
      content: reviewContent,
      likes: 0,
    });
```
and we avoid doing a second db call

Reply:
right, thanks for advice